package.json

```diff
@@ -41,7 +41,12 @@
     "eslint-plugin-react-hooks": "4.2.0",
     "eslint-plugin-standard": "5.0.0",
     "prettier": "2.3.2",
-    "stylelint": "13.13.1",
+    "stylelint": "13.13.1", 
+    "stylelint-config-standard": "21.0.0",
+    "typescript": "4.2.4"
+  },
+  "dependencies": {
```

Comment:
our main dependencies are handled in the base of the repo, though it seems this was done to add the package `use-force-update`. we should hopefully be able to use the core react package- what issue were you running into?

Reply:
I was not running into an issue but a forceUpdate function was used but I could not see where it came from. Actually, it was used as props according to my understanding now. I have removed the dependency. Thanks